parlai/tasks/glue/agents.py

```diff
@@ -7,6 +7,14 @@
 from parlai.tasks.huggingface.agents import AbstractHuggingFaceTeacher
 
 
+class AxTeacher(AbstractHuggingFaceTeacher):
+    hf_path = 'glue'
+    hf_name = 'ax'
+    hf_text_fields = ['premise', 'hypothesis']
+    hf_label_field = 'label'
+    hf_splits_mapping = {'train': 'test', 'valid': 'test', 'test': 'test'}
```

Comment:
This is an evaluation dataset (see https://huggingface.co/datasets/glue) so it only has a test split. 
An alternative is to have train and valid map to None.

Reply:
Good call out. Not sure what's going to happen if we make an empty teacher but we can try